query/gen_insert_statements_test.go

```diff
@@ -192,8 +192,8 @@ SELECT pk(), * FROM test;
   "a": "a",
   "b": "b",
   "c": {
-    "c": 1,
-    "d": 2
+    "c": 1.0,
```

Comment:
@asdine that's required because otherwise, you don't get the exact type and end up with a failing test. Due to the recursive nature of the structure that we're comparing, that's the tradeoff we have to make to get nice diffs. If we used a custom equality function, we'd go back to square one with large diffs. 

It sounds acceptable to me, but I wanted to run it past you before merging it.